tests/web/rpc/v1/test_endpoint_get_trace.py

```diff
@@ -35,6 +35,25 @@
 ) - timedelta(minutes=180)
 _SPAN_COUNT = 120
 _REQUEST_ID = uuid.uuid4().hex
+_NORMALIZED_FIELDS = [
+    "organization_id",
+    "project_id",
+    "service",
+    "trace_id",
+    "span_id",
+    "parent_span_id",
+    "segment_id",
+    "segment_name",
+    "is_segment",
+    "_sort_timestamp",
+    "start_timestamp",
+    "end_timestamp",
+    "duration_micro",
+    "exclusive_time_micro",
+    "name",
+    "sampling_factor",
+    "sampling_weight",
+]
```

Comment:
The fact that you re copying this list in the test and in the code should give you pause